travis/before-cache-gradle

```diff
@@ -5,3 +5,9 @@ caches="$HOME/.gradle/caches"
 rm -rf \
    "$caches/modules-2/modules-2.lock" \
    "$caches"/*/plugin-resolution
+
+find "$caches" -name '*.lock' -delete
```

Comment:
Sort these multiple `find` commands alphabetically. Consider fusing into a single `find` command, but OK to skip that if it looks like it will be significantly less readable.

Reply:
I sorted them, putting `*` commands at the end